util/config/config.go

```diff
@@ -87,6 +86,8 @@ const (
 )
 
 var (
+	SyncPruning      = true
+	LivePruning      = false
```

Comment:
So that `LivePruning` option no longer configurable?

Reply:
It was not configurable before. We will only make it configurable if we are sure it will not cause any problem